docs/proposals/job_controller.md

```diff
@@ -0,0 +1,191 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller, which will be responsible
+for managing pods that require to run:
+* once on demand or at a specified point in time
+* repeatedly at a specified point in time.
+
+Several exiting issues were already created regarding that particular subject:
+* Distributed CRON jobs [#2156](https://github.com/GoogleCloudPlatform/kubernetes/issues/2156)
+* Job Controller [#1624](https://github.com/GoogleCloudPlatform/kubernetes/issues/1624)
+
+
+## Use Cases
+1. Ability to have a controller that will create a run-once pod.
```

Comment:
Shouldn't this be "Ability to have a controller that will create a series of run-once pod(s)" - I think plurality of pods is key to the use case so it can support divide and conquer workflows?  I would expect the ability to implement a MapReduce should be covered as a key use case?